src/Arcus.WebApi.Security/Authorization/Jwt/JwtTokenReader.cs

```diff
@@ -39,21 +43,33 @@ public JwtTokenReader(string applicationId)
         /// </summary>
         /// <remarks>Uses Microsoft OpenId connect discovery endpoint</remarks>
         /// <param name="tokenValidationParameters">Collection of parameters to influence how the token validation is done</param>
-        public JwtTokenReader(TokenValidationParameters tokenValidationParameters) : this(tokenValidationParameters, MicrosoftDiscoveryEndpoint)
+        /// <param name="claimCheck">Custom claims key-value pair to validate against</param>
+        public JwtTokenReader(TokenValidationParameters tokenValidationParameters, IDictionary<string, string> claimCheck = null) : this(tokenValidationParameters, MicrosoftDiscoveryEndpoint, claimCheck)
         {
         }
 
+        /// <summary>
+        ///     Constructor
+        /// </summary>
+        /// <param name="applicationId">Azure AD Application used as audience to validate against</param>
+        public JwtTokenReader(string applicationId) : this(new Dictionary<string, string> {{ JwtClaimTypes.Audience, applicationId}})
+        {
+            Guard.NotNullOrWhitespace(applicationId, nameof(applicationId));
+        }
+
         /// <summary>
         ///     Constructor
         /// </summary>
         /// <param name="openIdConnectDiscoveryUri">Uri of an OpenId connect endpoint for discovering the configuration</param>
         /// <param name="tokenValidationParameters">Collection of parameters to influence how the token validation is done</param>
-        public JwtTokenReader(TokenValidationParameters tokenValidationParameters, string openIdConnectDiscoveryUri)
+        /// <param name="claimCheck">Custom claims key-value pair to validate against</param>
+        public JwtTokenReader(TokenValidationParameters tokenValidationParameters, string openIdConnectDiscoveryUri, IDictionary<string, string> claimCheck = null)
```

Comment:
We may consider creating an method overload instead of an optional parameter because it will result in `MethodNotFound` exceptions when the packages are updated seperatly.